scripts/irods/test/test_delay_queue.py

```diff
@@ -412,13 +455,15 @@ def test_sigpipe_in_delay_server(self):
                     self.admin,
                     parameters['metadata_attr'],
                     'You should never see this line.'))
+                lib.delayAssert(self.no_delayed_rules)
 
         finally:
             os.remove(rule_file)
-            self.admin.assert_icommand(['iqdel', '-a'])
+            self.admin.run_icommand(['iqdel', '-a'])
             irodsctl.restart()
 
     @unittest.skipIf(plugin_name == 'irods_rule_engine_plugin-python', 'Delete this line on resolution of #4094')
+    @unittest.skipIf(test.settings.TOPOLOGY_FROM_RESOURCE_SERVER, 'odbc.ini file does not exist on CSC')
```

Comment:
we don't use CSC in documentation anywhere... so consider more words here

Reply:
Done